server/index.ts

```diff
@@ -17,6 +18,10 @@ async function run(listenPort: number): Promise<void> {
         `${__dirname}/data/galapagotchi.db`,
         `${__dirname}/data/event-logs/events-${startTime}.log`,
     )
+    const lnRpc = await createLnRpc({
```

Comment:
Should probably have the ability to disable Lightning so that running Galapagotchi locally doesn't require a whole lightning node